src/SayMore/Model/ArchivingHelper.cs

```diff
@@ -185,9 +186,11 @@ private static void AddIMDISession(Session saymoreSession, ArchivingDlgViewModel
 			imdiSession.SocialContext = GetFieldValue(sessionFile, "additional_Social_Context");
 			imdiSession.Task = GetFieldValue(sessionFile, "additional_Task");
 
+			imdiSession.AddContentDescription(new LanguageString{Value = GetFieldValue(sessionFile, "notes"), Iso3LanguageId = analysisLanguage});
+
 			// custom session fields
 			foreach (var item in saymoreSession.MetaDataFile.GetCustomFields())
-				imdiSession.AddContentKeyValuePair(item.FieldId, item.ValueAsString);
+				imdiSession.AddContentKeyValuePair(item.FieldId.Substring(XmlFileSerializer.kCustomFieldIdPrefix.Length), item.ValueAsString);
```

Comment:
The commit comment mentions some specific fields having the 'custom_' removed, but this code change removes it for all custom fields in the session and the person field. Is that desired?